cmake/README.md

```diff
@@ -0,0 +1,27 @@
+# Experimental CMake support
+
+The files in this directory are mostly for dealing with dependencies.
+
+We use
+[ExternalProject](https://cmake.org/cmake/help/latest/module/ExternalProject.html)
+to download and build missing dependencies, during CMake's configuration time.
+
+Still TODO for CMake support:
+- Benchmarks.
+- Top-level examples other than helloworld.
+- Leaf examples (e.g. trace/examples/ dir).
+- Exporters other than stdout.
+- CI (Travis) support.
+- No shared library for now.
+
+## Quickstart:
+
+```shell
+cmake -H. -B.build
+cmake --build .build
+cmake --build .build --target test
```

Comment:
That only works with `make` as the generator, with things like `MSBuild` it does not.  This works with all generators:

```
(cd .build && ctest --output-on-failure)
```

Well, on Windows you need to add `-C Debug` or `-C Release` for "reasons".

Reply:
Thanks, I've updated the docs! I'm still pretty new to CMake.